srcpkgs/dracut/template

```diff
@@ -61,6 +61,7 @@ post_install() {
 
 	rm ${DESTDIR}/usr/share/man/man8/*.service.*
 	rm -r ${DESTDIR}/usr/lib/kernel
+	echo "DRACUT_VERSION=%{version}-%{revision}" > ${DESTDIR}/usr/lib/dracut/dracut-version.sh
```

Comment:
This seems more like an upstream release engineering task. The version ought to be encoded in `dracut-version.sh` in the source tarball. Is the official expectation really that distribution packagers should be setting version information?

Reply:
https://github.com/dracutdevs/dracut/issues/2169

Reply:
This doesn't work because the default Makefile target will create a `dracut-version.sh` with no version information, so the overwrite never happens. What seems to work is to define the template function

```bash
pre_build() {
    DRACUT_VERSION="${version}" make dracut-version.sh
}
```

that will force write the `dracut-version.sh` file before the package is built. I've pushed a fixup commit on your branch that makes this change. Please review and, if acceptable, squash the commits back down.l